src/client/datascience/editor-integration/cellhashprovider.ts

```diff
@@ -535,7 +537,9 @@ export class CellHashProvider implements ICellHashProvider {
 
         const fileMatch = /^File.*?\[\d;32m(.*):\d+.*\u001b.*\n/.exec(traceFrame);
         if (fileMatch && fileMatch.length > 1) {
-            const fileUri = Uri.file(fileMatch[1]);
+            // We need to untilde the file path here for the link to work in VS Code
```

Comment:
Wasn't in the milestone, but by the time I investigate this customer bug it was basically a one line fix, so checked in it.